.github/workflows/release.yml

```diff
@@ -12,6 +12,8 @@ jobs:
       - uses: actions/setup-node@v1
         with:
           node-version: 12
+      - run: yarn install
+      - run: yarn build
```

Comment:
Also for release a build is necessary. semantic-release should commit and push the artefacts to master.